### ISSUE TITLE
Defeated tribes share vision in 3+ team games

### ISSUE THREAD

**Issue by jonathanmclaus** [island-troll-tribes/island-troll-tribes#329]
This is likely due to converting the defeated tribe to observer players, which shares vision.

**Comment by Jaccouille** [comment]
I don't believe this issue is still relevant, we did multiple 3+ team game yet I never had shared vision when one team was defeated.

**Comment by jonathanmclaus** [comment]
We should test this with 3 players. I don't recall why exactly this is a problem, as defeated tribes can't really do anything anyway. I guess it would allow base defenses to attack people only another base can see.

**Comment by Jaccouille** [comment]
I ran a test with 3 game clients, setting 3 players in 1v1v1, 1 player with buildings had been defeated, the building still belonged to the tribe player but didn't share vision until the end game state, I believe this issue can be closed.
